### ISSUE TITLE
Bug: the changed data will be lost when page is switched in properties window

### ISSUE THREAD

**Issue by 0x5bfa** [files-community/Files#14506]
### Description

If you changed properties in a certain tab, that changes will be lost after you switched tabs.

### Steps To Reproduce

1. Go to folder properties window
2. Go to Customization tab
3. Change icon without tapping Save button
4. Go to General tab
5. Go back to Customization tab

Your icon selected is no longer selected.

### Requirements

I think it'd be better to instance all ViewModels of properties tab and put them into a list and use them every time frame content is changed instead of instancing new one.

### Files Version

3.0.0

### Windows Version

10.0.22000

### Log File

No logs because it's not unexpected behavior but implemented behavior.

**Comment by yaira2** [comment]
Thanks for the report

**Comment by yaira2** [comment]
Would caching the pages help?

**Comment by 0x5bfa** [comment]
What I am thinking is caching view models in `MainPropertiesViewModel` and use them when navigated.
I have this idea and this is blocked by not having view models for each (and being low priority).
